jstests/core/explain_use_backup_plan.js

```diff
@@ -0,0 +1,28 @@
+// Test that the explain will use backup plan if the original winning plan ran out of memory in the
+// "executionStats" mode
+// This test was designed to reproduce SERVER-32721"
+(function() {
+    "use strict";
+
+    db.foo.drop() let bulk = db.foo.initializeUnorderedBulkOp();
+    for (let i = 0; i < 100000; ++i) {
```

Comment:
This test is great, but we tend to favor tests that don't need as much data to be inserted, since they will execute faster. In this case you configure the maxBlockingSortBytes, so do you need this many documents?